files/image_config/monit/conf.d/sonic-host

```diff
@@ -31,6 +31,13 @@ check program routeCheck with path "/usr/local/bin/route_check.py"
     every 5 cycles
     if status != 0 for 3 cycle then alert repeat every 1 cycles
 
+# dualtor_neighbor_check.py: script to check if the neighbor entries in APPL_DB
+# has correct neighbor or tunnel route entries in ASIC_DB based on the mux
+# states.
+check program dualtorNeighborCheck with path "/usr/local/bin/dualtor_neighbor_check.py -o SYSLOG -s ERROR"
```

Comment:
looking at routeCheck above, there is no `-o` or `-s `option. Why do we need to explicitly set here?

Reply:
Those arguments are to set the log output destination(`-o`) and log level(`-s`)